src/PHPCR/NamespaceRegistryInterface.php

```diff
@@ -32,6 +32,11 @@
     const PREFIX_JCR = "jcr";
 
     /**
+     * A constant for the predefined namespace prefix "rep".
+     */
+    const PREFIX_REP = "rep";
```

Comment:
Ok, it really seems to be up do the implementation which node type to use... Then this one is obsolete.